contribute/merge-pull-request.md

```diff
@@ -0,0 +1,116 @@
+# Merge a pull request
```

Comment:
Should we add a section about draft PRs? Is it OK to open a draft PR, should maintainers review drafts, etc

Reply:
right I was looking for a section header somewhere that wasn't about merging that I could add this suggestion to :joy: